src/api/propensity-api.js

```diff
@@ -263,38 +83,21 @@ export let PropensityScore;
  *         enum listed in the Event enum above.
  *
  *  @typedef {{
- *    name: string,
+ *    name: !Event,
  *    active: boolean,
  *    data: ?JsonObject,
  * }}
  */
 export let PropensityEvent;
 
+export const Event = LoggerApi.Event;
+export const SubscriptionState = LoggerApi.SubscriptionState;
+
 /**
+ * @extends {LoggerApi.LoggerApi}
  * @interface
  */
 export class PropensityApi {
```

Comment:
Don't have to delete lines 277:to 297, you can simply at @override to lines 288 and 295

Reply:
Why re-write the same exact thing with an override?

Reply:
two reasons: 
1. completeness, when we extend interfaces, overriding existing interfaces is a common practice and it documents the new interface fully
2. looks like in this case even that might not work since, the argument to sendEvent() and sendSubscriptionState() now uses PropensityApi which is a clone to LoggerAPI but not the same. Does the closure allow you to send PropensityApi.Event when the method its overriding wants LoggerApi.Event?

Reply:
Yes because they are identical.  If I change one then it breaks though.

Reply:
I've added those functions back now.